### DISCUSSION TITLE
Welcome to rust-cheat-checker Discussions!

### DISCUSSION THREAD

**Discussion by Rend-y** [Rend-y/rust-cheat-checker#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by REDaHEAD223** [comment]
> ## 👋 Welcome!
> We’re using Discussions as a place to connect with other members of our community. We hope that you:
> 
> * Ask questions you’re wondering about.
> * Share ideas.
> * Engage with other community members.
> * Welcome others and are open-minded. Remember that this is a community we
>   build together 💪.
> 
> To get started, comment below with an introduction of yourself and tell us about what you do with this community.

How can I contact you? Discord links are not working.

**Reply by Rend-y** [reply]
Hi my discord: midoriya4333
And I update discord link 
[click](https://discord.com/invite/X7hhuENts9)
